install.sh

```diff
@@ -127,7 +127,8 @@ else
     apt-get install -y sudo wget build-essential unzip git zstd iputils-ping lsb-release aptitude bubblewrap jq ${axel_inst}
 fi
 
-LOGDIR="/var/lib/pacstall/metadata"
+METADIR="/var/lib/pacstall/metadata"
+LOGDIR="/var/log/pacstall/error_log"
 SCRIPTDIR="/usr/share/pacstall"
 PACDIR="/tmp/pacstall"
 PACSTALL_USER=$(logname 2> /dev/null || echo "${SUDO_USER:-${USER}}")
```

Comment:
since you redeclared this in update, could you also fix this to use the `whoami`?

`PACSTALL_USER=$(logname 2> /dev/null || echo "${SUDO_USER:-${USER:-$(whoami)}}")`